mutis/lib/correlation.py

```diff
@@ -312,10 +312,10 @@ def _comp_nb(tis, dtis):
 # MC estimators of uncertainties
 #
 
-# def MC_corr_err(t1, s1, t1p, t2, s2, t2p, t, dt, fcorr, fgen, N=400):
+# def MC_corr_err(t1, s1, t1p, t2, s2, t2p, t, dt, fcorr, fgen, n=400):
```

Comment:
Is all this commented out text expected to be part of the code? Otherwise I'd suggest to remove it

Reply:
Yep, I agree..
there are other functions that are not used like `lib.correlation.fkroedel` and `lib.correlation.fwelsh` that are not used either and that I would also remove, for the moment..

Reply:
done

Reply:
Yes these MC_... are not used anymore, I commented them long ago, they can be removed completely.